packages/_private/angular-demo/src/demoform/demoform.component.html

```diff
@@ -0,0 +1,211 @@
+<form
+  [formGroup]="formData"
+  (reset)="reset()"
+  (ngSubmit)="submit(formData)"
+  (syn-change)="synChange($event)"
+  #form
+>
+  <!-- PersonalInformation -->
+  <demo-fieldset legend="Personal Information">
+
+    <syn-radio-group
+      formControlName="gender"
+      id="radiogroup-gender"
+      name="gender"
+      label="Please tell us your gender"
+      [required]="true"
+    >
+      <syn-radio value="f">Female</syn-radio>
+      <syn-radio value="m">Male</syn-radio>
+      <syn-radio value="other">Other</syn-radio>
+    </syn-radio-group>
+
+    <syn-select
+      formControlName="role"
+      id="select-role"
+      label="Current position"
+      name="role"
+      [required]="true"
+      v-model="formData.role"
+    >
+      <syn-optgroup label="Developers">
+        <syn-option value="backend">Backend Developer</syn-option>
+        <syn-option value="frontend">Frontend Developer</syn-option>
+      </syn-optgroup>
+      <syn-optgroup label="Other">
+        <syn-option value="lead">Team Lead</syn-option>
+        <syn-option value="other">Other (please specify in comment section below)</syn-option>
+      </syn-optgroup>
+    </syn-select>
+
+    <syn-input
+      formControlName="name"
+      id="input-text"
+      label="Name"
+      [minlength]="5"
+      [maxlength]="20"
+      name="name"
+      placeholder="Please insert a value for the regular text input (between 5 and 20 Characters)"
+      [required]="true"
+      type="text"
+    />
+
+    <syn-input
+      formControlName="email"
+      id="input-email"
+      label="E-Mail"
+      name="email"
+      placeholder="Please insert your E-mail address"
+      [required]="true"
+      type="email"
+    />
+
+    <syn-input
+      formControlName="phone"
+      id="input-phone"
+      label="Phone"
+      name="phone"
+      placeholder="Please provide your phone number"
+      [required]="true"
+      type="tel"
+    />
+
+    <syn-input
+      formControlName="date"
+      id="input-date"
+      label="Date of birth"
+      name="date"
+      placeholder="Please insert your E-mail address"
```

Comment:
Should be removed. Is the message from email and a placeholder is not needed for date anyway. This needs to be fixed in all wrappers